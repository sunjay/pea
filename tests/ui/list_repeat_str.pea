fn main() {
    let a = [3; b"abc"];
}
