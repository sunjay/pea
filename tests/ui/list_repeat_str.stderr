stack backtrace:
   0: main
error: list repeat length argument must be a number, found `[u8]`
